systemtest/src/main/java/io/strimzi/systemtest/resources/crd/KafkaResource.java

```diff
@@ -124,6 +124,7 @@ private static KafkaBuilder defaultKafka(Kafka kafka, String name, int kafkaRepl
                 .editKafka()
                     .withVersion(Environment.ST_KAFKA_VERSION)
                     .withReplicas(kafkaReplicas)
+                    .addToConfig("log.message.format.version", Environment.ST_KAFKA_VERSION.substring(0, 3))
```

Comment:
That will bite you at some point!

Reply:
what do you suggest? Add second env variable for log.message.format or create method which will return it base don kafka version?

Reply:
Personally I would parse it properly. I think you have enough env vars in your life.

Reply:
I was looking into code and probably the best way for this is parsing the value from `kafka-versions.yaml`. This will need some additional changes to `TestKafkaVersion` so I would keep this for another PR ( I can open it afterward). Do you agree with it?

Reply:
OK